docs/language/learn-ql/writing-queries/query-help.rst

```diff
@@ -84,7 +84,8 @@ The following elements are optional child elements of the ``section``, ``example
    |                | | ``height`` Optional, height of the image.              |                    |                                                                                                                                                                                                                                                                                                           |
    |                | | ``width`` Optional, the width of the image.            |                    |                                                                                                                                                                                                                                                                                                           |
    +----------------+----------------------------------------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-   | ``include``    | ``src`` The query help file to include.                  | None               | Include a query help file at the location of this element. See :ref:`Query help inclusion <qhelp-inclusion>` below for more information.                                                                                                                                                                  |
+   | ``include``    | ``src`` The query help file to include.                  | None               | Include a query help file at the location of this element. See :ref:`Query help inclusion <
+   -inclusion>` below for more information.                                                                                                                                                                  |
```

Comment:
```suggestion
   | ``include``    | ``src`` The query help file to include.                  | None               | Include a query help file at the location of this element. See :ref:`Query help inclusion <qhelp-inclusion>` below for more information.                                                                                                                                                                  |
```

Sorry, I didn't spot the weird newline when reviewing!

Reply:
Thanks - maybe I introduced it in the conflict resolution.